scripts/ecdsa/mock/deploy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# -x echos all lines for debug
+# set -x
+
+export FOUNDRY_DISABLE_NIGHTLY_WARNING=1
+
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+# Function to display error message and exit
+error_exit() {
+    echo "Error: $1" >&2
+    exit 1
+}
+
+# Set RPC URLs based on environment
+if [ "$DEPLOY_ENV" = "TESTNET" ]; then
+    MOCK_RPC_URL=${MOCK_RPC_URL:-}
+else
+    MOCK_RPC_URL=${MOCK_RPC_URL:-http://localhost:8546}
+fi
+
+# Validate RPC URLs
+if [ -z "$MOCK_RPC_URL" ]; then
+    error_exit "MOCK_RPC_URL is not set in the environment variables (tip: grab from .nodes/avs_deploy.json)"
+fi
+
+# Get chain ID from rpc url
+MOCK_CHAIN_ID=$(cast chain-id --rpc-url "$MOCK_RPC_URL") || error_exit "Failed to get chain ID from RPC URL"
+export MOCK_CHAIN_ID
+echo "Chain ID: $MOCK_CHAIN_ID"
+
+# Read the deployer private key from file
```

Comment:
We don't need this (as this is mainly for the source chain eigenlayer deploy)

I'd just generate a random key and throw it away.

But check with @dakom maybe he wants to modify these mocks later. In which case I'd recommend taking a private key as an env arg (and if missing, just generate one for deploy and don't save it anywhere)